nni/retiarii/oneshot/pytorch/sampling.py

```diff
@@ -0,0 +1,339 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import random
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+
+from nni.retiarii.nn.pytorch.api import LayerChoice, InputChoice
+from .random import PathSamplingLayerChoice, PathSamplingInputChoice
+from .base_lightning import BaseOneShotLightningModule
+
+
+class StackedLSTMCell(nn.Module):
```

Comment:
If nothing is changed for `StackedLSTMCell` and `ReinforceController`, I think you can just import them from `.enas`.